rllib/tuned_examples/pendulum-sac.yaml

```diff
@@ -0,0 +1,37 @@
+# Pendulum SAC can attain -150+ reward in 5-6k timesteps (*disregarding initial exploration steps like in softlearning codebase)

```

Comment:
I think it's generally better to include the exploration steps in the sample count, i.e. just add 1k to those numbers.

Reply:
Fixed!